### ISSUE TITLE
Error: Could not locate the bindings file. Tried: ui/node_modules/deasync/build/deasync.node

### ISSUE THREAD

**Issue by rajaaliya2020** [abbr/deasync#98]
* Electron Version: ^2.0.0
* Operating System (Platform and Version): Mac High Seira

**Expected Behavior**
The binding should work. The node app (express) works when run outside the electron 

**Actual behavior**
 Error: Could not locate the bindings file. Tried: ui/node_modules/deasync/build/deasync.node

**Comment by grantcv1** [comment]
I have been having this same problem for ages now. The instructions say to run 'npm install deasync' but, on the Windows 10 platform, at least, that is not adequate.

After I have installed deasync, the resulting deasync folder contains bin and src folder and a number of files. Their is no build folder and nothing get built. There are, however, numerous (about 50) OS/Node folders within the bin folder, each one containing a copy of the missing file.

When I attempt to require deasync into my application, the system looks all over the place for deasync.node file -- in build, build/Debug, build/Release, build/default, out/Debug, out/Release, Debug,  Release, and finally compiled/8.2.1/win32/x64. None of these locations exist. What does exist is bin/win32-x64-node-8, but it doesn't look there. I've always just manually copied the file into a location the system looks, but it seems I'm either missing a step, or there is a bug. This has happened for me on two computers and for everyone else who has tried to setup my application into a dev environment.

**Comment by abbr** [comment]
This is a issue of electron and deasync doesn't officially support electron. For reasons, see #81 or search other issues using keyword electron.

**Comment by grantcv1** [comment]
FWIW, I completely redesigned my app to use JavaScript's async/await feature and removed deasync from my stack. Made my life a bit easier.

**Comment by markhughes** [comment]
This happens from bitbucket pipelines if it runs a test that includes a service that depends on it 


```
    Could not locate the bindings file. Tried:
     → /opt/atlassian/pipelines/agent/build/node_modules/deasync/build/deasync.node
     → /opt/atlassian/pipelines/agent/build/node_modules/deasync/build/Debug/deasync.node
     → /opt/atlassian/pipelines/agent/build/node_modules/deasync/build/Release/deasync.node
     → /opt/atlassian/pipelines/agent/build/node_modules/deasync/out/Debug/deasync.node
     → /opt/atlassian/pipelines/agent/build/node_modules/deasync/Debug/deasync.node
     → /opt/atlassian/pipelines/agent/build/node_modules/deasync/out/Release/deasync.node
     → /opt/atlassian/pipelines/agent/build/node_modules/deasync/Release/deasync.node
     → /opt/atlassian/pipelines/agent/build/node_modules/deasync/build/default/deasync.node
     → /opt/atlassian/pipelines/agent/build/node_modules/deasync/compiled/12.16.2/linux/x64/deasync.node
     → /opt/atlassian/pipelines/agent/build/node_modules/deasync/addon-build/release/install-root/deasync.node
     → /opt/atlassian/pipelines/agent/build/node_modules/deasync/addon-build/debug/install-root/deasync.node
     → /opt/atlassian/pipelines/agent/build/node_modules/deasync/addon-build/default/install-root/deasync.node
     → /opt/atlassian/pipelines/agent/build/node_modules/deasync/lib/binding/node-v72-linux-x64/deasync.node
```

**Comment by vp2177** [comment]
This is now happening again, using node `v14.0.0` on Mac.
@abbr please consider re-opening.

**Comment by papb** [comment]
@vp2177 Node v14 had some regressions involving streams, which are being fixed, probably v14.0.1 will be released soon, I'm not sure if this is causing your problem but let's wait a bit and see if node 14.0.1 will fix it for you :)

**Comment by abbr** [comment]
This should be fixed by v0.1.20, which has pre-compiled binaries for node 14.
